Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:3.9-alpine
+
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR app/
+
+COPY requirements.txt requirements.txt
+RUN pip install --no-cache-dir -r requirements.txt
+
+COPY . .
+
+CMD ["python", "app/main.py"]
```

Comment:
Ensure that the path `app/main.py` is correct. The `CMD` command should point to the correct location of your main script. Verify that `main.py` is located in the `app` directory.